src/api/index.js

```diff
@@ -626,6 +628,43 @@ const API = {
   },
 };
 
+async function loadCountyCitiesMap() {
+  return new Promise((resolve, reject) => {
+    const options = {
+      header: true,
+      error(err) {
+        reject(err);
+      },
+    };
+    const dataStream = request.get(
+      'https://docs.google.com/spreadsheets/d/1jNk90L1FGXt3estVzFII2-eeKQZ85RYiLKCyNe14nGg/export?format=csv&gid=0'
+    );
+    const parseStream = Papa.parse(Papa.NODE_STREAM_INPUT, options);
+    dataStream.pipe(parseStream);
```

Comment:
Can you may be comment (or add a link to specific doc section) why we need to do all these somersaults.